python/tvm/target/detect_target.py

```diff
@@ -58,6 +58,17 @@ def _detect_rocm(dev: Device) -> Target:
     )
 
 
+def _detect_opencl(dev: Device) -> Target:
+    return Target(
+        {
+            "kind": "opencl",
+            "max_shared_memory_per_block": dev.max_shared_memory_per_block,
+            "max_threads_per_block": dev.max_threads_per_block,
+            "thread_warp_size": dev.warp_size,
```

Comment:
What about other attributes which are specified in `target_kind.cc`?

Reply:
The rest of attributes will use the default value.  meta, cuda, and rocm are similar ways to initialize.